### PR TITLE
[WIPTEST] Add template fleecing

### PR DESCRIPTION
**Pull request by dajohnso**
{{pytest: -v -k '(power_control and not distributed) or (analysis and not host and not datastore)' --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7c4f1941145732086351b206cccbb3281e214cca:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c193cc2a285e148edbc1a4bf053a1528590f9be5:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4c0e2bad5626c320a8fb9809eff87d362d571645:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit fb1238cedde5c439e142e129bbbc6b0c6d872eb4:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 850d6d98b5d0be139d012073251f754203df6255:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1ff62bbd1053fcee22bb13c3ab93dba502894c3d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please?
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 16a92e4d507268c467eaf441be99479ffccc284f:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./utils/appliance.py:361:5: E303 too many blank lines (2)
```
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1ab9716a29dbb1677f230ab133b08e7794a36209:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./utils/appliance.py:361:5: E303 too many blank lines (2)
```
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: utils/appliance.py [hunk 1]
```diff
@@ -416,6 +425,22 @@ def version(self):
     def log(self):
         return create_sublogger(self.address)
 
+    def appliance_object(self):
```
Comment (seandst) [inline utils/appliance.py]: I bet this is slow as hell...

I've been threatening to make the multithreaded-multiprovider-method-doer prototyped in [scripts/sync_template_tracker.py](https://github.com/RedHatQE/cfme_tests/blob/master/scripts/sync_template_tracker.py#L24) a generic thing that we can use elsewhere, and this probably a good use-case for it.

**Comment by vrutkovs** [comment]
Right, I'll write a new test to check templates SSA, based on existing test_instance_SSA

**Comment by psav** [comment]
@dajohnso You OK with us closing this on off in preference of the newer one?

**Comment by psav** [comment]
Closing this in favour of @vrutkovs new solution

**Pull request closed without merging**